static-build/CMakeLists.txt

```diff
@@ -53,6 +53,11 @@ set(DEPENDENCY_CFLAGS "${DEPENDENCY_CFLAGS} ${HARDENING_FLAGS}")
 set(DEPENDENCY_CXXFLAGS "${DEPENDENCY_CXXFLAGS} ${HARDENING_FLAGS}")
 set(DEPENDENCY_LDFLAGS "${DEPENDENCY_LDFLAGS} ${HARDENING_LDFLAGS}")
 
+option(DEV_BUILD "Enable build options that make debugging more convenient" OFF)
+set_prefix_map_flags()
+set(DEPENDENCY_CFLAGS "${DEPENDENCY_CFLAGS} ${MACRO_PREFIX_MAP_FLAG} ${DEBUG_PREFIX_MAP_FLAGS}")
+set(DEPENDENCY_CXXFLAGS "${DEPENDENCY_CXXFLAGS} ${MACRO_PREFIX_MAP_FLAG} ${DEBUG_PREFIX_MAP_FLAGS}")
```

Comment:
Why do we need it here? AFAIU we're only interested in stripping file names for the tarantool project so propagating `DEV_BUILD` to `ExternalProject_Add(tarantool)` below would be enough.